tools/PowerShell-DSC/DatadogApmDotnet.ps1

```diff
@@ -0,0 +1,60 @@
+Configuration DatadogApmDotnet
+{
+    param
+    (
+        # Target nodes to apply the configuration
+        [string]$NodeName = 'localhost',
+
+        # Determines whether to install the Agent
+        [bool]$InstallAgent = $true,
+
+        # Determines whether to install the Tracer
+        [bool]$InstallTracer = $true
+    )
+
+    Import-DscResource -ModuleName PSDscResources -Name MsiPackage
+    Import-DscResource -ModuleName PSDscResources -Name Environment
+
+    # Version of the Agent package to be installed
+    $AgentVersion = '7.27.0'
+
+    # Version of the Tracer package to be installed
+    $TracerVersion = '1.26.1'
+
+    Node $NodeName
+    {
+        # Agent msi installer
+        if ($InstallAgent) {
+            MsiPackage 'dd-agent' {
+                Path      = "https://s3.amazonaws.com/ddagent-windows-stable/ddagent-cli-$AgentVersion.msi"
+                ProductId = 'B55FFED6-0CAD-4F94-AA07-5B74A5776C1C'
+                Ensure    = 'Present'
+            }
+        }
+
+        # .NET Tracer msi installer
+        if ($InstallTracer) {
+            MsiPackage 'dd-trace-dotnet' {
+                Path      = "https://github.com/DataDog/dd-trace-dotnet/releases/download/v$TracerVersion/datadog-dotnet-apm-$TracerVersion-x64.msi"
+                ProductId = '00B19BDB-EC40-4ADF-A73F-789A7721247A'
+                Ensure    = 'Present'
+            }
+
+            Environment 'COR_PROFILER' {
```

Comment:
Why do we want to set the environment variable here instead of sticking with the .NET Tracer MSI defaults?

Reply:
You are right. I was experimenting with the idea of doing as much as possible during installation so that the app deployment only needs to set one env var (`COR_ENABLE_PROFILING`) instead of two. My thinking was that the app itself only needs to know if it wants to enable tracing or not, and shouldn't worry about getting the profiler GUID right.

This idea could extend to the msi itself, too. We could set `COR_PROFILER` globally so there's one less env vars for users to set when hosting outside of IIS.

But it was just an experiment, so we can remove this.